plugin.js

```diff
@@ -1,17 +1,17 @@
 'use strict'
 
 const fastq = require('fastq')
-const EE = require('events').EventEmitter
-const inherits = require('util').inherits
+const EventEmitter = require('events')
 const debug = require('debug')('avvio')
 const { AVV_ERR_READY_TIMEOUT } = require('./lib/errors')
 
 // this symbol is assigned by fastify-plugin
 const kPluginMeta = Symbol.for('plugin-meta')
+function noop () {}
 
 function getName (func, optsOrFunc) {
   // use explicit function metadata if set
-  if (func[kPluginMeta] && func[kPluginMeta].name) {
+  if (func[kPluginMeta]?.name) {
```

Comment:
Because of `?.` is used, node version < 12 is not supported after this pr be merged